packages/sdk/src/index.ts

```diff
@@ -28,6 +28,12 @@ export interface HasId<I> {
 export interface Callable {
   (...args: any[]): any;
 }
+
+export type SerializableEvent = Omit<
```

Comment:
this is needed because we cant easily make full `Event` types from `Logs` or `LogDescription`, this type is compatible with `Event`, and `Logs & LogDescription` types